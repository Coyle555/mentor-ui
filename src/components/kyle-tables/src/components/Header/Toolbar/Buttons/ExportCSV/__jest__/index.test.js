import React from 'react';
import { ExportCSV } from '../index';
import renderer from 'react-test-renderer';

test('Export CSV button default render', () => {
	const tree = renderer.create(<ExportCSV csvURL="" />).toJSON();

	expect(tree).toMatchSnapshot();
});

test('Export CSV disabled button', () => {
	const tree = renderer.create(<ExportCSV disabled={true} csvURL="" />).toJSON();

	expect(tree).toMatchSnapshot();
});

test('Export CSV with a url', () => {
	const tree = renderer.create(<ExportCSV csvURL="foo" />).toJSON();

	expect(tree).toMatchSnapshot();
});
