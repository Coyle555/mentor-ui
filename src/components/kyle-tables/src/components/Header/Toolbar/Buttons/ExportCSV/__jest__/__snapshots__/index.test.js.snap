// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Export CSV button default render 1`] = `
<a
  className="btn-table"
  data-for="table-tooltip"
  data-tip="Export To CSV"
  download={true}
  href=""
>
  <i
    className="fal fa-file-csv"
  />
</a>
`;

exports[`Export CSV disabled button 1`] = `
<a
  className="btn-table"
  data-for="table-tooltip"
  data-tip="Export To CSV"
  disabled={true}
  download={true}
  href=""
>
  <i
    className="fal fa-file-csv"
  />
</a>
`;

exports[`Export CSV with a url 1`] = `
<a
  className="btn-table"
  data-for="table-tooltip"
  data-tip="Export To CSV"
  download={true}
  href="foo"
>
  <i
    className="fal fa-file-csv"
  />
</a>
`;
